.github/workflows/contract-testing.yaml

```diff
@@ -29,6 +29,7 @@ jobs:
      stream-instance: ${{ steps.filter.outputs.stream-instance }}
      search-deployment: ${{ steps.filter.outputs.search-deployment }}
      alert-configuration: ${{ steps.filter.outputs.alert-configuration }}
+     resource-policy: ${{ steps.filter.outputs.resource-policy }}
```

Comment:
knit: in a following PR it would be great to have tests sorted alphabetically as in TF